server/datastore/mysql/migrations/tables/20231215122713_InsertPolicyStatsData.go

```diff
@@ -0,0 +1,81 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231215122713, Down_20231215122713)
+}
+
+func Up_20231215122713(tx *sql.Tx) error {
+	// Update Counts for Inherited Global Policies for each Team
```

Comment:
copy/pasta from https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/policies.go

Reply:
Am ok with copy pasta.

Let's add a `// NOTE ...` reference in this code to the source. And add a `// NOTE ...` reference from the source to here.

Rationale: If the original query or this one needs a bug fix in the future then developers will be aware that they need to fix the other query too.